fiftyone/operators/types.py

```diff
@@ -1378,6 +1378,23 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
 
+class LazyFieldView(View):
+    """Displays a lazy text input which only apply input field changes on blur
+    or when user clicks the save button within the field.
+
+    .. note::
+
+        Must be used with :class:`String` or :class:`Number` properties.
+
+    Args:
+        disable_save_on_blur (None): prevents applying field changes on blur.
```

Comment:
nit: no strong preference but could consider `save_on_blur=False` to be more concise, with default being `save_on_blur=True`

Reply:
Just lurking unwanted here but @imanjra as a plug-in dev i have no idea what blur means in this context, would be helpful to explain in the docstring